cost_index/tests/test_cost_index_api.py

```diff
@@ -55,3 +61,110 @@ def test_cost_index_viewset(
+
+
+@pytest.mark.django_db
+def test_get_apartment_revaluation_summary_unauthorized(
+    api_client, user_api_client, profile_api_client, drupal_salesperson_api_client
+):
+    response = api_client.get(reverse("cost_index:apartment-revaluation-summary"))
+    assert response.status_code == 403
+
+    response = user_api_client.get(reverse("cost_index:apartment-revaluation-summary"))
+    assert response.status_code == 403
+
+    response = profile_api_client.get(
+        reverse("cost_index:apartment-revaluation-summary")
+    )
+    assert response.status_code == 403
+
+    response = drupal_salesperson_api_client.get(
+        reverse("cost_index:apartment-revaluation-summary")
+    )
+    assert response.status_code == 403
+
+
+@pytest.mark.django_db
+def test_get_apartment_states(
+    drupal_server_api_client, elastic_haso_project_with_5_apartments
+):
+    response = drupal_server_api_client.get(
+        reverse("cost_index:apartment-revaluation-summary")
+    )
+    assert response.status_code == 200
+    assert response.data == []
+
+    _, haso_apartments = elastic_haso_project_with_5_apartments
+
+    haso_0 = haso_apartments[0]
+    haso_0_reservation_0 = ApartmentReservationFactory(
+        apartment_uuid=haso_0.uuid,
+        state=ApartmentReservationState.CANCELED,
+        list_position=1,
+    )
+    haso_0_reservation_1 = ApartmentReservationFactory(
+        apartment_uuid=haso_0.uuid,
+        state=ApartmentReservationState.CANCELED,
+        list_position=2,
+    )
+
+    haso_1 = haso_apartments[1]
+    haso_1_reservation_0 = ApartmentReservationFactory(
+        apartment_uuid=haso_1.uuid,
+        state=ApartmentReservationState.CANCELED,
+        list_position=1,
+    )
+
+    haso_2 = haso_apartments[2]
+    haso_2_reservation_0 = ApartmentReservationFactory(
+        apartment_uuid=haso_2.uuid,
+        state=ApartmentReservationState.CANCELED,
+        list_position=1,
+    )
+
+    now = timezone.now()
+    with mock.patch("django.utils.timezone.now") as mock_now:
+        mock_now.return_value = now - timedelta(hours=2000)
+        haso_0_revaluation_0 = ApartmentRevaluationFactory(
+            apartment_reservation=haso_0_reservation_0,
+        )
+    with mock.patch("django.utils.timezone.now") as mock_now:
+        mock_now.return_value = now - timedelta(minutes=20)
+        haso_0_revaluation_1 = ApartmentRevaluationFactory(
+            apartment_reservation=haso_0_reservation_1,
+        )
+    with mock.patch("django.utils.timezone.now") as mock_now:
+        mock_now.return_value = now - timedelta(minutes=5)
+        haso_1_revaluation_0 = ApartmentRevaluationFactory(
+            apartment_reservation=haso_1_reservation_0,
+        )
+    with mock.patch("django.utils.timezone.now") as mock_now:
+        mock_now.return_value = now - timedelta(minutes=61)
+        ApartmentRevaluationFactory(
+            apartment_reservation=haso_2_reservation_0,
+        )
```

Comment:
There is also [freezegun lib](https://github.com/spulec/freezegun) (already used in this project) for setting the current time in tests.

Reply:
awesome!